dashboard/modules/reporter/reporter_agent.py

```diff
@@ -31,6 +31,10 @@
 
 # Are we in a K8s pod?
 IN_KUBERNETES_POD = "KUBERNETES_SERVICE_HOST" in os.environ
+# Flag to enable showing disk usage when running in a K8s pod,
+# disk usage defined as the result of running psutil.disk_usage("/")
+# in the Ray container.
+ENABLE_K8S_DISK_USAGE = os.environ.get("ENABLE_K8S_DISK_USAGE", False)
```

Comment:
Ray specific flags should be prefixed with Ray -- maybe `RAY_DASHBOARD_ENABLE_K8S_DISK_USAGE"?

Reply:
Yep, fixed!